index.bs

```diff
@@ -5294,38 +5294,40 @@ partial interface MLGraphBuilder {
     // of the input values itself, in order to increase the numerical stability of
     // the result.
     // [1]: https://cs231n.github.io/linear-classify/#softmax
-    const max_x = builder.reduceMax(x, { axes: [1], keepDimensions: true });
+    const max_x = builder.reduceMax(x, { axes: [axis], keepDimensions: true });
     const exp_x = builder.exp(builder.sub(x, max_x));
-    return builder.div(exp_x, builder.reduceSum(exp_x, { axes: [1], keepDimensions: true }));
+    return builder.div(exp_x, builder.reduceSum(exp_x, { axes: [axis], keepDimensions: true }));
   </pre>
 </details>
 </div>
 
-#### {{MLGraphBuilder/softmax(input)}} #### {#api-mlgraphbuilder-softmax-input}
+#### {{MLGraphBuilder/softmax(input, axis)}} #### {#api-mlgraphbuilder-softmax-input-axis}
 <div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input 2-D tensor.
+        - *axis*: an {{unsigned long}} scalar. The dimension the softmax will be performed on.
```

Comment:
🤔 What do you think? The softmax operation as a whole is actually applied to all dimensions, not just one, but the reduction *aspect* of softmax is applied to only one. Should we say:

```suggestion
        - *axis*: an {{unsigned long}} scalar. The dimension the reduction will be performed on.
```

Or:
```suggestion
        - *axis*: an {{unsigned long}} scalar. The dimension the softmax reduction will be performed on.
```

No change, because I'm being pedantic:
```suggestion
        - *axis*: an {{unsigned long}} scalar. The dimension the softmax will be performed on.
```